# The define below does not fit well with LegacySupport one;
# however, we require LegacySupport so far for getline.

--- src/util/rsdnsutils.cc.orig	2023-02-19 03:51:45.000000000 +0800
+++ src/util/rsdnsutils.cc	2023-03-04 09:45:22.000000000 +0800
@@ -33,19 +33,19 @@
 #include <netdb.h>
 #endif
 
-#ifdef __APPLE__
-#include <AvailabilityMacros.h>
-#if __MAC_OS_X_VERSION_MIN_REQUIRED < 1070
-static size_t strnlen(const char *s, size_t maxlength) {
-  size_t l = 0;
-  while (l < maxlength && *s) {
-    l++;
-    s++;
-  }
-  return l;
-}
-#endif
-#endif
+//#ifdef __APPLE__
+//#include <AvailabilityMacros.h>
+//#if __MAC_OS_X_VERSION_MIN_REQUIRED < 1070
+//static size_t strnlen(const char *s, size_t maxlength) {
+//  size_t l = 0;
+//  while (l < maxlength && *s) {
+//    l++;
+//    s++;
+//  }
+//  return l;
+//}
+//#endif
+//#endif
 
 //https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-2
 constexpr uint16_t DNSC_IN    = 1; //Internet (IN)
